src/app.html

```diff
@@ -17,7 +17,7 @@
     <a class="nav-link" id="nav-about" href="#/about">About</a>
     <a class="nav-link open" id="nav-header" href="#/header">Header</a>
     <a class="nav-link" id="nav-mime" href="#/mime">MIME types</a>
-    <a class="nav-link" id="nav-security" href="#/security">Security</a>
+    <a class="nav-link" id="nav-security-definitions" href="#/securityDefinitions">Security</a>
```

Comment:
The URL directly maps to the form names.
I added the missing security requirements form and changed the URL to `#/global-security`.